package.json

```diff
@@ -21,7 +21,9 @@
     "lodash": "^4.17.19",
     "nconf": "^0.10.0",
     "node-fetch": "^2.1.2",
-    "pino": "^4.17.3"
+    "pino": "^4.17.3",
+    "tinyurl": "^1.1.7",
+    "yargs-parser": "13.1.2"
```

Comment:
Are we still using `yargs-parser` anywhere, or is this holdover from a different iteration of this pull request?

Reply:
This was supposed to be a dependency upgrade. Dependabot told me this was out of date, however now I see I may have to actually update the yarn lock file to resolve to a newer version. 😔